app/models/xkcd.rb

```diff
@@ -0,0 +1,9 @@
+# Xkcd model for getting Xkcd images
+class Xkcd < ApplicationRecord
```

Comment:
This class does not need to inherit from `ApplicationRecord`, it should be a plain Ruby class:

`class Xkcd`